locales/en.json

```
@@ -185,12 +185,21 @@
             "description": "Please review your order, or try again later."
         },
         "card": {
-            "invalid_number": "The card's number is invalid.",
+            "invalid_number": "The card's number is invalidThe card's number is invalid..",
             "invalid_date": "The card's expiration date is invalid.",
             "invalid_year": "The card's expiration year is invalid.",
             "invalid_month": "The card's expiration month is invalid.",
             "invalid_cvv": "The card's CVV is invalid.",
-            "expired": "The card's expiration date is passed"
+            "expired": "The card's expiration date is passed.",
+            "declined": "The card has been declined.",
+            "address_invalid": "The card’s address is incorrect. Check the card’s address or use a different card.",

```

J'aurais tendance a enlever la 2ieme phrase vu qu'on l'écrit pas pour les autres erreurs simples.
```suggestion
            "address_invalid": "The card's address is incorrect.",

```